proxy/logging/LogObject.cc

```diff
@@ -983,7 +998,7 @@ LogObjectManager::_solve_filename_conflicts(LogObject *log_object, int maxConfli
           if (logfile.open_file() == LogFile::LOG_FILE_NO_ERROR) {
             long time_now = LogUtils::timestamp();
 
-            if (logfile.roll(time_now - log_object->get_rolling_interval(), time_now) == 0) {
+            if (logfile.roll(time_now - log_object->get_rolling_interval(), time_now)) {
```

Comment:
Looking at LogFile::roll(),  return value of 1 means successful rotation? And this line should be looking for the error condition

Reply:
Oh, great catch did not mean to change this line, likely did not restore this line properly after my experiments.